Examples/VoiceMemos/VoiceMemos/AudioRecorderClient/LiveAudioRecorderClient.swift

```diff
@@ -1,6 +1,5 @@
 import AVFoundation
 import ComposableArchitecture  // TODO: Should `UncheckedSendable` live in `Dependencies`?
-import Foundation
```

Comment:
Note: This was added at Concurrency Beta (https://github.com/pointfreeco/swift-composable-architecture/pull/1189)